atd-vzv/src/views/summary/FatalitiesMultiYear.js

```diff
@@ -261,98 +261,53 @@ const FatalitiesMultiYear = () => {
-          </Row>
+        <Col xs="6" sm="4" md="2">
+          <h6>Prior Years:</h6>
         </Col>
-        <Col md="2">
-          <Row>
-            <Col md="12">
-              <h6>
-                <strong>{calculateYearlyTotals(lastYearDeathArray)}</strong>
-              </h6>
-            </Col>
-          </Row>
-          <Row>
-            <Col md="12">
-              <h6>in {getYearsAgoLabel(1)}</h6>
-            </Col>
-          </Row>
+        <Col xs="6" sm="4" md="2">
+          <h6>
+            <strong>{calculateYearlyTotals(lastYearDeathArray)}</strong> in{" "}
+            {getYearsAgoLabel(1)}
+          </h6>
         </Col>
-        <Col md="2">
-          <Row>
-            <Col md="12">
-              <h6>
-                <strong>{calculateYearlyTotals(twoYearsAgoDeathArray)}</strong>
-              </h6>
-            </Col>
-          </Row>
-          <Row>
-            <Col md="12">
-              <h6>in {getYearsAgoLabel(2)}</h6>
-            </Col>
-          </Row>
+        <Col xs="6" sm="4" md="2">
+          <h6>
+            <strong>{calculateYearlyTotals(twoYearsAgoDeathArray)}</strong> in{" "}
+            {getYearsAgoLabel(2)}
+          </h6>
         </Col>
-        <Col md="2">
-          <Row>
-            <Col md="12">
-              <h6>
-                <strong>
-                  {calculateYearlyTotals(threeYearsAgoDeathArray)}
-                </strong>
-              </h6>
-            </Col>
-          </Row>
-          <Row>
-            <Col md="12">
-              <h6>in {getYearsAgoLabel(3)}</h6>
-            </Col>
-          </Row>
+        <Col xs="6" sm="4" md="2">
+          <h6>
+            <strong>{calculateYearlyTotals(threeYearsAgoDeathArray)}</strong> in{" "}
+            {getYearsAgoLabel(3)}
+          </h6>
         </Col>
-        <Col md="2">
-          <Row>
-            <Col md="12">
-              <h6>
-                <strong>{calculateYearlyTotals(fourYearsAgoDeathArray)}</strong>
-              </h6>
-            </Col>
-          </Row>
-          <Row>
-            <Col md="12">
-              <h6>in {getYearsAgoLabel(4)}</h6>
-            </Col>
-          </Row>
+        <Col xs="6" sm="4" md="2">
+          <h6>
+            <strong>{calculateYearlyTotals(fourYearsAgoDeathArray)}</strong> in{" "}
+            {getYearsAgoLabel(4)}
+          </h6>
         </Col>
-        <Col md="2">
-          <Row>
-            <Col md="12">
-              <h6>
-                <strong>{calculateYearlyTotals(fiveYearsAgoDeathArray)}</strong>
-              </h6>
-            </Col>
-          </Row>
-          <Row>
-            <Col md="12">
-              <h6>in {getYearsAgoLabel(5)}</h6>
-            </Col>
-          </Row>
+        <Col xs="6" sm="4" md="2">
+          <h6>
+            <strong>{calculateYearlyTotals(fiveYearsAgoDeathArray)}</strong> in{" "}
+            {getYearsAgoLabel(5)}
+          </h6>
         </Col>
       </Row>
+      <br></br>
```

Comment:
what is the `<br>` tag for? Seems like a shortcut that should really be solved with margin/padding utility classes or overall formatting and theming

Reply:
Good call - fixed!